app/controllers/labels_controller.rb

```diff
@@ -0,0 +1,48 @@
+class LabelsController < ApplicationController
+  before_action :set_label, only: [:edit, :update, :destroy]
+
+  def new
+    @label = Label.new
+  end
+
+  def edit
+  end
+
+  def create
+    @label = Label.new(label_params)
+
+    respond_to do |format|
+      if @label.save
+        format.html { redirect_to @label, notice: 'Label was successfully created.' }
+      else
+        format.html { render :new }
+      end
+    end
+  end
```

Comment:
respond_toを記載する必要が無いように思われます。gem jbuilderを有効化していると、rails g コマンド実行時に生成されると思うので、不要であればgem自体をコメントアウトか削除して良いと思います。
```suggestion
  def create
    @label = Label.new(label_params)
    if @label.save
      redirect_to @label, notice: 'Label was successfully created.'
    else
      render :new
    end
  end
```